web/src/app/pages/edit-survey/edit-survey.component.spec.ts

```diff
@@ -105,10 +135,32 @@ describe('EditSurveyComponent', () => {
       fixture.detectChanges();
     }));
 
-    it('displays survey id in title', () => {
-      const tempTitle = fixture.debugElement.query(By.css('#temp-title'))
-        .nativeElement.innerText;
-      expect(tempTitle).toContain(surveyId);
+    it('displays left menu items with router link', () => {
+      const surveyButton = fixture.debugElement.query(By.css('#survey-button'))
```

Comment:
Nit: Could you use queryAll and map to get these labels and links with a little less repetition?

Or parametrizing could also be the way to go. Internally we have the concept of "pit" (parametrized it), though not sure if that exists out in the wild. There's also the old fashioned way of parametrizing: https://jdkandersson.com/2019/06/30/parametrize-angular-jasmine-tests/

But I do feel that there are too many expects here for one test. I think it would be cleaner to have separate it blocks for each of the links, one way or another.

Reply:
Done. I like `pit`, it looks neat, thanks!